stan/math/rev/mat/functor/cvodes_ode_data.hpp

```diff
@@ -53,12 +68,37 @@ class cvodes_ode_data {
                   const std::vector<T_param>& theta,
                   const std::vector<double>& x, const std::vector<int>& x_int,
                   std::ostream* msgs)
-      : y0_(y0),
+      : f_(f),
+        y0_(y0),
         theta_(theta),
+        theta_dbl_(value_of(theta)),
         N_(y0.size()),
         M_(theta.size()),
-        param_var_ind_(initial_var::value ? N_ : 0),
-        ode_system_(f, value_of(theta), x, x_int, msgs) {}
+        x_(x),
+        x_int_(x_int),
+        msgs_(msgs),
+        S_((initial_var::value ? N_ : 0) + (param_var::value ? M_ : 0)),
+        coupled_ode_(f, y0, theta, x, x_int, msgs),
+        coupled_state_(coupled_ode_.initial_state()),
+        nv_state_(N_VMake_Serial(N_, &coupled_state_[0])),
+        nv_state_sens_(NULL),
```

Comment:
we're now using C++11! Want to change this `NULL` to `nullptr`? (If there's other changes being made on this branch; if not, we can hold off.)

Reply:
Changed it.